doc/containers.md

```diff
@@ -172,7 +172,7 @@ host\_name              | string    | randomly assigned | no        | bridged, p
 hwaddr                  | string    | randomly assigned | no        | all                           | -             | The MAC address of the new interface
 mtu                     | integer   | parent MTU        | no        | all                           | -             | The MTU of the new interface
 parent                  | string    | -                 | yes       | physical, bridged, macvlan    | -             | The name of the host device or bridge
-vlan                    | integer   | -                 | no        | macvlan                       | network\_vlan | The VLAN ID to attach to
+vlan                    | integer   | -                 | no        | macvlan, physical             | network\_vlan | The VLAN ID to attach to
```

Comment:
The case you describe is not specific to VLAN case.
If you have two containers, one attached to eth0 using macvlan nictype and the other attached to eth0 using physical nictype, you now need to start them in the right order. First the "macvlan" container, then the "physical" one, otherwise the main physical device will be gone from the host and you can't create the macvlan device.
Actually the problem is not vlan or not vlan. The problem is mixing a "physical" NIC with NICs derived the the physical one regardless the nictype is macvlan, macvlan+vlan or physical+vlan.